en/sensor/pmw3901.md

```diff
@@ -42,15 +42,15 @@ The PX4 team mainly tested using the [Lidar Lite V3](../sensor/lidar_lite.md) on
 <span id="orientation"></span>
 ## Mounting/Orientation
 
-Flow modules are typically mounted near the centre of the vehicle.
+Flow modules are typically mounted near the center of the vehicle.
 If mounted off-centre you will need to set offsets: [Optical Flow > EKF2](../sensor/optical_flow.md#ekf2).
 
-The flow modules may be mounted with any yaw orientation relative to vehicle body frame, but you must set the value used in [SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT).
+The flow modules may be mounted with any yaw orientation relative to the vehicle body frame, but you must set the value used in [SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT).
```

Comment:
Previous text was correct and this is incorrect. It must be either "to vehicle body frame" or "to the vehicle's body frame". The former is more concise.

Reply:
Actually, I am not sure on this. Leaving as is.